libtenzir/builtins/contexts/lookup_table.cpp

```diff
@@ -527,6 +666,7 @@ class ctx final : public virtual context {
 private:
   map_type context_entries;
   detail::subnet_tree subnet_entries;
+  duration update_time_;
```

Comment:
This also needs to be persisted, otherwise the update timeout will stop functioning after a restart of the node.